drupal_test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -x
+set -e
+
+export
+
+composer create-project --no-interaction drupal-composer/drupal-project:8.x-dev drupal
+cd drupal
+composer config repositories.fabpot vcs https://github.com/fabpot/Twig
```

Comment:
you should not add your own fork as a repository here. That won't work for PR builds (and will run the wrong code for branch builds in the main repo). You should instead add a `path` repository pointing to the local Twig clone instead (the one available here). This would probably require making this `drupal` folder a sibling instead of a child though

Reply:
Good idea, I will do that.